examples/incremental_pca.py

```diff
@@ -0,0 +1,54 @@
+# Copyright Contributors to the Cellarium project.
+# SPDX-License-Identifier: BSD-3-Clause
+
+"""
+Example: Incremental PCA
+========================
+
+This example shows how to fit feature count data to incremental PCA
+model [1, 2].
+
+Example run::
+    python examples/incremental_pca.py fit \
+        --model.module scvid.module.IncrementalPCAFromCli \
+        --data.filenames "gs://dsp-cellarium-cas-public/test-data/benchmark_v1.{000..003}.h5ad" \
+        --data.shard_size 10_000 --data.max_cache_size 2 --data.batch_size 10_000 \
+        --data.num_workers 4 \
+        --trainer.accelerator gpu --trainer.devices 1 --trainer.default_root_dir runs/ipca \
+        --trainer.callbacks scvid.callbacks.ModuleCheckpoint
+
+**References:**
+
+    1. *A Distributed and Incremental SVD Algorithm for Agglomerative Data Analysis on Large Networks*,
+       M. A. Iwen, B. W. Ong
+       (https://users.math.msu.edu/users/iwenmark/Papers/distrib_inc_svd.pdf)
+    2. *Incremental Learning for Robust Visual Tracking*,
+       D. Ross, J. Lim, R.-S. Lin, M.-H. Yang
+       (https://www.cs.toronto.edu/~dross/ivt/RossLimLinYang_ijcv.pdf)
+"""
+
+from lightning.pytorch.cli import LightningCLI
+
+from scvid.data import DistributedAnnDataCollectionDataModule
+from scvid.train.training_plan import TrainingPlan
+
+
+class IncrementalPCACLI(LightningCLI):
```

Comment:
The naming clash between `IncrementalPCAFromCli` in `scvid/module` and `IncrementalPCACLI` here makes the code difficult to read. How about renaming `IncrementalPCACLI` in here (`examples/incremental_pca.py`) to `_LightningCLIWithLinks`?